examples/solverdummy/solverdummy.py

```diff
@@ -20,56 +20,60 @@
 participant_name = args.participantName
 
 if participant_name == 'SolverOne':
-    write_data_name = 'dataOne'
-    read_data_name = 'dataTwo'
-    mesh_name = 'MeshOne'
+    write_data_name = 'Data-One'
+    read_data_name = 'Data-Two'
+    mesh_name = 'SolverOne-Mesh'
```

Comment:
Maybe move this into another commit/PR? Again: Not 100% related to this update and makes this PR bigger than necessary.